src/theme/uiColors.ts

```diff
@@ -1,10 +1,29 @@
-import { ThemeContext } from "../types";
+import { CatppuccinWorkbenchMode, ThemeContext } from "../types";
 import { opacity, transparent, shade } from "./utils";
 
+const getWorkbenchColors = (context: ThemeContext) => {
+  const { palette, options } = context;
+
+  const workbenchMode = options.workbenchMode;
+  const colorsMap: Record<CatppuccinWorkbenchMode, Record<string, string>> = {
+    default: {},
+    flat: {
+      "breadcrumb.background": palette.base,
+      "editorGroupHeader.tabsBackground": palette.mantle,
+      "sideBarSectionHeader.background": palette.mantle,
+      "sideBarTitle.background": palette.mantle,
+      "tab.border": palette.base,
```

Comment:
This would make reduce even more places where `crust` is used, resulting in a more flat, 2-level-shade appearance.

```suggestion
      "activityBar.background": palette.mantle,
      "breadcrumb.background": palette.base,
      "commandCenter.background": palette.mantle,
      "debugToolBar.background": palette.mantle,
      "editorGroupHeader.tabsBackground": palette.mantle,
      "minimap.background": opacity(palette.base, 0.5),
      "sideBarSectionHeader.background": palette.mantle,
      "sideBarTitle.background": palette.mantle,
      "statusBar.background": palette.mantle,
      "statusBar.noFolderBackground": palette.mantle,
      "tab.border": palette.base,
      "titleBar.activeBackground": palette.mantle,
      "titleBar.inactiveBackground": palette.mantle,
```